src/pages/Dashboard/Investment.tsx

```
@@ -1,55 +1,34 @@
-			completionDate: "2023-05-01",
-			status: "Pending",
-		},
+		" ",
 	];
+	
+
+	
+	const [selectedItem, setSelectedItem] = useState<Data | null>(null);
+	const [showModal, setShowModal] = useState(false)
+
+	const { isLoading, data } = useFetchUserInvestmentQuery({}) as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)